File: Java/src/main/java/org/sisuns/bucles_while/basico/Ejercicio_02.java
package org.sisuns.bucles_while.basico;

public class Ejercicio_02 {
    public static void main(String[] args) {
        // Declaración e inicialización de la variable numero
        int numero = 1;

        // Mientras el número sea menor o igual a 20
        while (numero <= 20) {
            // Mostrar el número actual
            System.out.print(numero);

            // Si el número es divisible por 5, mostrar 0
            if (numero % 5 == 0) {
                System.out.print(0);
            }

            // Incrementar el número
            numero = numero + 1;
        }
    }
}

File: Python/Bucles_while/Basico/Ejercicio_02.py
# Inicialización del número
numero = 1

# Mientras el número sea menor o igual a 20
while numero <= 20:
    # Mostrar el número actual
    print(numero, end=" ")

    # Si el número es divisible por 5, mostrar 0
    if numero % 5 == 0:
        print(0, end=" ")

    # Incrementar el número
    numero += 1
